client/src/app/management/components/member-edit/member-edit.component.ts

```diff
@@ -130,6 +129,7 @@ export class MemberEditComponent extends BaseModelContextComponent implements On
                         const title = user.getTitle();
                         super.setTitle(title);
                         this.user = user;
+                        setTimeout(() => (this.isEditingUser = this.route.snapshot.url[0]?.path === 'edit'));
```

Comment:
That is dirty. No inner subscription has happened, so replace `getUserByUrl` with observing the route.

Reply:
btw, the behavior you see occurs because you are moving between URLs in the very same component, thus angular does not re-run `ng onInit`